staging/src/k8s.io/client-go/tools/clientcmd/api/v1/types.go

```diff
@@ -214,6 +214,12 @@ type ExecConfig struct {
 	// present. For example, `brew install foo-cli` might be a good InstallHint for
 	// foo-cli on Mac OS systems.
 	InstallHint string `json:"installHint,omitempty"`
+
+	// ProvideClusterInfo determines whether or not to provide cluster information,
+	// which could potentially contain very large CA data, to this exec plugin as a
+	// part of the KUBERNETES_EXEC_INFO environment variable. By default, it is set
+	// to false.
```

Comment:
can point to the helper package with methods for dealing with the envvar here

Reply:
https://github.com/kubernetes/kubernetes/pull/95489/commits/bd4af0df52b7da2a838795ccf9a8760a0316da48